images/miq-app/Dockerfile

```diff
@@ -3,6 +3,7 @@ MAINTAINER ManageIQ https://github.com/ManageIQ/manageiq-appliance-build
 
 ## Set build ARGs
 ARG REF=master
+ARG GHORG=ManageIQ
```

Comment:
I think this will break the clone for `manageiq-appliance` if you provide a REF that doesn't exist on both your fork of `manageiq` and the ManageIQ fork of `manageiq-appliance`.